tests/etl/conftest.py

```diff
@@ -3,40 +3,37 @@
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 
 import pytest
+import json
 from webtest.app import TestApp
+from urllib2 import HTTPError
 
-from treeherder.etl.mixins import OAuthLoaderMixin
-from treeherder.etl.oauth_utils import OAuthCredentials
 from treeherder.webapp.wsgi import application
 
-from treeherder.client import TreeherderClient
-from tests.sampledata import SampleData
+from treeherder import client
 
 
 @pytest.fixture
-def mock_post_json_data(monkeypatch, jm):
-    def _post_json_data(url, data):
+def mock_post_json_data(monkeypatch, set_oauth_credentials):
```

Comment:
Rewrote this fixture because the old one mocked too much.  We needed to mock lower down the chain so we were testing more of the code-path.